src/main/object-uploader.js

```diff
@@ -58,18 +58,17 @@ export default class ObjectUploader extends Transform {
   }
 
   _transform(chunk, encoding, callback) {
-    // Calculate the md5sum.
-    let md5 = Crypto.createHash('md5')
-    md5.update(chunk)
-    let md5sum = md5.digest()
-
     let method = 'PUT'
     let headers = {
       'Content-Length': chunk.length,
-      'Content-Type': this.contentType,
-      'Content-MD5': md5sum.toString('base64')
+      'Content-Type': this.contentType
     }
 
+    // Calculate md5 hash if SHA256 is not calulcated.
+    let md5sum = Crypto.createHash('md5').update(chunk).digest().toString('base64')
```

Comment:
Wouldn't this call be always called regardless of the value of `enableSHA256` ?

Reply:
MD5 is also being used in another place in the code. It seems to used as part of multi-part upload. I need to modify the comment to reflect that. I also need to modify the code to do just the digest instead of converting it to base64.